### PR TITLE
feat: merge upstream template v1.6.1 [PS-1038]

### PR DESCRIPTION
**Pull request by brad-getpassport**

### CODE REVIEW

File: .github/workflows/release-notifications.yaml [hunk 1]
```diff
@@ -9,9 +9,17 @@ permissions: {}
 jobs:
   slack:
     name: Slack release notification
+    # Remove the next line after your Slack app has been set up. See also:
+    # https://github.com/slackapi/slack-github-action#setup-2
+    if: ${{ false }}
```
Comment (jens-getpassport) [inline .github/workflows/release-notifications.yaml:14]: Do we want to keep this and remove in our actual package repos, or remove it in the template too?

File: .github/workflows/scorecards-analysis.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  - cron: 27 20 * * 1
+  push:
+    branches: [main]
+
+# Declare default permissions as read only.
+permissions: read-all
+
+jobs:
+  analysis:
+    name: Scorecards analysis
```
Comment (jens-getpassport) [inline .github/workflows/scorecards-analysis.yaml:15]: I think we need to take extra steps run [run the Scorecard Action on a private repo](https://github.com/marketplace/actions/ossf-scorecard-action#installation).

**Review by jens-getpassport — COMMENTED** [review]
Looks good, but can we wait until PR https://github.com/jenstroeger/python-package-template/pull/211 is merged? That’ll simplify the Actions.

**Comment by brad-getpassport** [comment]
@jens-getpassport yeah we can wait until you are finished with all your changes.

File: .github/workflows/release-notifications.yaml (hunk 1 above)
Comment (jens-getpassport) [reply .github/workflows/release-notifications.yaml:14]: Keep it in this template repo, remove the line in our actual repos.

File: .github/workflows/check-and-release-main.yaml [hunk 3]
```diff
@@ -1,50 +1,53 @@
...
-        github_token: ${{ secrets.GITHUB_TOKEN }}
+        github_token: ${{ secrets.CI_TOKEN }}
         pypi_token: ''  # We don't publish to PyPi: ${{ secrets.PYPI_TOKEN }}
+
+  publish:
+    runs-on: ubuntu-latest
+    needs: release
+    env:
+      PASSPORT_CI_TOKEN: ${{ secrets.CI_TOKEN }}
+      PASSPORT_CONFIG: servicer/getpassport-github.ini
```
Comment (jens-getpassport) [inline .github/workflows/check-and-release-main.yaml:46]: I think for the template repo we don’t have a `.ini` file, so we can maybe leave this value empty? Having an empty `PASSPORT_CONFIG` reminds us to set it in the actual repos.

**Review by jens-getpassport — APPROVED** [review]
Schnufte 👍

**Pull request merged**